<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fsk_config</class>
 <widget class="QDialog" name="fsk_config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>142</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>131</width>
     <height>85</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>recv gain</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_fsk_gain"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_fsk_config">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>129</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Save And Close</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>131</width>
     <height>85</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>tx gain</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_fsk_tx_gain"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
